src/sinks/loki/config.rs

```diff
@@ -93,8 +93,8 @@ pub struct LokiConfig {
     ///
     /// Both keys and values are templateable, which enables you to attach dynamic labels to events.
     ///
-    /// Labels can be suffixed with a `*` to allow the expansion of objects into multiple labels,
-    /// see [Label expansion][label_expansion] for more information.
+    /// Labels can be `*` or prefixes ending with `*` to allow the expansion of objects into
+    /// multiple labels, see [Label expansion][label_expansion] for more information.
```

Comment:
Not strictly related to your change but it strikes me that the following verbage may add clarity:

```suggestion
    /// Valid label keys include `*`,  and prefixes ending with `*`, to allow for the expansion of
    /// objects into multiple labels. See [Label expansion][label_expansion] for more information.
```

Reply:
Updating this and regenerating docs

Reply:
Resolved in 93cbfb6a9